problem-set/Sum1N.md

```diff
@@ -0,0 +1,83 @@
+# Sum1N
+
+პრობლემა:
+```
+მომხმარებელს შეყავს მთელი რიცხვი n, პროგრამამ უნდა დაბეჭდოს 1 დან n მდე რიცხვების
+ჯამი
+
+```
+
+
+
+## პრობლემის გადაჭრის გზა:
+ამოცანის ამოსახსნელად, საჭიროა, 3 ეტაპი:
+ *მომხმარებელმა უნდა შემოიტანოს რიცხვი n;
+ *პროგრამამ უნდა დათვალოს ჯამი 1 დან n მდე ნატურალური რიცხვებისა.
+ *პროგრამამ უნდა დაპრინტოს ინფორმაცია კონსოლში.
+
+
+
+
+---
+
+### I.მომხმარებლის მიერ შემოტანილი ინფორმაციის წაკითხვა:
+ * მომხმარებელმა კონსოლში უნდა შემოიტანოს რიცხვი n, რასაც ვაკეთებთ readInt() მეთოდის მეშვეობით. იმისათვის, რომ ჩვენს კონსოლის პროგრამას გასაგები ინტერფეისი ჰქონდეს, readInt()-მეთოდს ფრჩხილებში გადავცემთ არგუმენტს, ანუ იმას თუ რა უნდა დაპრინტოს პროგრამამ, ვიდრე მომხმარებელი შემოიყვანს რიცხვს (n-ს). საბოლოოდ მივიღებთ readInt("Enter n: "); სინტაქსს. readInt მეთოდი გვიბრუნებს იმ რიცხვის (int-ტიპის) მნიშვნელობას, რომელიც მომხმარებელმა შემოიტანა და, შესაბამისად, ვინახავთ მას num ცვლადში. შესაბამისად, მივიღეთ კოდი : int num = readInt("Enter n: ");
+
+### II.ჯამის დათვლა 1 დან n მდე ნატურალური რიცხვებისა:
+ * ცხადია, ჯამის დათვლა რამდენიმენაირად შეგვიძლია, თუმცა ყველაზე მარტივი და ინტუიტიური მაინც ფორმულის გამოყენებაა მათემატიკიდან. 1 დან n-მდე რიცხვების ჯამი მოიცემა ფორმულით ( n*(n+1) ) / 2 -ს, რომლის გამოყვანაც შეგვიძლია არითმეტიკული პროგრესიის ჯამის ფორმულიდან -> (A(1)+A(n))*n /2. ხოლო ჩვენ შემთხვევაში n-ის მაგივრად გვქენება num-ცვლადი და შესაბამისად მიღებულ მნიშვნელობას შევინახავთ sumFormula ცვლადში.
```

Comment:
ამ ამოხსნის კოდიც ეწეროს ერთერთ ვარიანტად, მაგრამ მნიშვნელოვანი მაინც for ციკლით ამოხსნაა